src/borg/remote.py

```diff
@@ -202,16 +328,96 @@ def write(self, fd, to_send):
         return written
 
 
+def api(*, since, **kwargs_decorator):
+    """Check version requirements and use self.call to do the remote method call.
+
+    since specifies the version in which borg introduced this method,
```

Comment:
I think in Borg we mostly used either the olde Sphinx style (those ugly ":param foo: foo is bar" blocks) or the `*foo* is the bar to ...` style.

Personally I very much prefer the "write a complete sentence mentioning the parameters block" method.